Update links to example configuration profiles

tux234 opened the pull request:
Fixing broken links in the article to point to absolute paths.

tux234 committed:
Update links to example configuration profiles

tux234 committed:
Update links to example configuration profiles

nonpunctual reviewed (changes requested):
@tux234 I would argue that these things (even if they are linked in articles) in the solutions dir.

https://github.com/fleetdm/fleet/tree/main/docs/solutions

tux234 committed:
Move Okta configuration profiles to docs and update article links

Move example configuration profiles from assets/configuration-profiles/ to
docs/solutions/macos/configuration-profiles/ to align with existing documentation
structure. Update all profile links in the Okta Platform SSO article to reference
the new location.

tux234 committed:
Merge configuration profile updates into main PR branch

Merge changes from tux234-article-okta_psso-fix-links branch, bringing in
the moved configuration profile files and updated article links.

tux234 committed:
Move Okta configuration profiles to docs and update article links

Move example configuration profiles from assets/configuration-profiles/ to
docs/solutions/macos/configuration-profiles/ to align with existing documentation
structure. Update all profile links in the Okta Platform SSO article to reference
the new location.

tux234 commented:
Good call @nonpunctual!

I didn't know that folder existed. I moved the files underneath that folder and updated the links. Looks like @ddribeiro needs to approve adding files under docs/solutions.

@ddribeiro, 

Please let me know if this is the correct spot, or if you have any questions.

nonpunctual reviewed (approved):
LGTM

nonpunctual committed:
Merge branch 'main' into tux234-article-okta_psso-fix-links-1
